celery_progress/backend.py

```diff
@@ -1,26 +1,28 @@
 from abc import ABCMeta, abstractmethod
 from decimal import Decimal
 
+from asgiref.sync import async_to_sync
 from celery.result import AsyncResult
+from channels.layers import get_channel_layer
```

Comment:
I wonder if we could inline these imports so that people who don't need websockets don't need to install `asgiref` and `channels`. what do you think?

Reply:
As you stated, this could probably be added as a separate branch of the project, or as a different project entirely. That would eliminate the extra dependencies if people plan to use http only.